### ISSUE TITLE
Remixed + HP WMR headset : Black screen, sound only

### ISSUE THREAD

**Issue by kakek** [LibreVR/Revive#1148]
When launching games with revive/remixedn the game seems to start, I can hear the sound of the game, but the haedset and the screen remain black.
Game works fine under steamVR revive, but has performance issues.
The game is the climb, though it seems to be the same with every game.

  - CPU: I5 3570K OCed
  - GPU: GTX 1060 6GO
  - RAM: 16GO DDR3
  - OS: Windows 10 17763.193 (Latest normal version I think. I opted out of insider preview for this test
  - WMR headset / HP

SteamVR is not concerned, since I'm trying to use remixed.

I tried revive 1.6 with remixed, and 1.5 with the same result.

When launching the game, I juste start mixed reality portal, then revive dashboard (with steam turned off.) and clic on the game ... nothing else.

Really hope you can help me. It's not only this game, I'd really like to have access to the oculus library with decent performances.

Thanks for the awesome work on revive anyway !

**Comment by Dragoboss** [comment]
Is it completely black or do you get the loading animation with the white balls circling each other?
Occasionally it happens it takes a while for your pc to load a game into the headset. Only occasionally. Usually it works fine for me.

Has it worked at all in the past?
Or did you only recently get your HMD?

I wouldn't know how to reproduce this. For me most things load fine.

**Comment by kakek** [comment]
Behold, as I revive this issue 3 months later to continue the discussion.

So, to answer your questions : 
I never tried to use remixed before. I used revive a bit before remixed existed. With steamVR it still works for most games. But as I said performance are below actually playable ( 55-60 FPS for the climb, makes for a bad experience. Other game lack a low FPS indicator, but I can tell I'm not runnning a smooth 90fps at all.)

I tried with other games, and the exact behavior varies, but I never make it ingame.

Dreamdeck or crow are stuck at the white loading balls.
The climb seems to start (as I said, music and sound effect from the menu play ) but it's all black in the headset.
Epic roller coaster start on the computer screen (I can even use mouse to interact) but the headset is all black, and movement of the headset don't translate ingame.
Brass tactics also goes to the totaly black headset, but it also says on the computer screen that I need paired touch controlers.

All in one, it would seems that the headset is simply not detected.

Also, it's worth mentioning that I on a windows insider release.
